### DISCUSSION TITLE
How to make acap app run at startup on power-up?

### DISCUSSION THREAD

**Discussion by AmarRobo** [AxisCommunications/acap-native-sdk-examples#112] [General] [answered]
I am trying the object detection app, the app gets turned off on restarting the camera. The axis apps like object analytics or video motion detection remains running while the object detection app does not. Is there any option to make the object detection keep on running ?

**Comment by Corallo** [comment, marked as answer]
Hello @amararjun 

Are you using the [object detection acap example from the native sdk examples repository](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/object-detection), or the [object detection in the computer vision sdk examples repository](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python)?

In the first case, you can add `respawn` as runMode in the manifest of your acap. You can check the documentation [here](https://axiscommunications.github.io/acap-documentation/docs/develop/manifest-schemas/schema-field-descriptions-v1.3.1.html)

In the second case, if your application is a container, you could try what was suggested in this [issue](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/issues/17)

**Reply by AmarRobo** [reply]
Hi @Corallo 

Thank you! I am using the native sdk examples. I posted on the wrong repository. Thanks.

**Reply by Corallo** [reply]
No worries, I'm transferring it to the right repo.
